adder.js

```diff
@@ -1,6 +1,6 @@
 function validate(input) {
```

Comment:
As I can see "Input" function parameter not using. 
Why are you still passing it?

Reply:
Please look at issue #11 as a possible better solution. 
And explain as a comment to the next pull request, why it will be better than your solution.